### DISCUSSION TITLE
1.9 UI improvement

### DISCUSSION THREAD

**Discussion by the-w41k3r** [pterodactyl/panel#4172] [Feature Requests]
Thank you for the amazing new update. Although the new UI looks cool, the Start, Stop and Restart buttons look a bit off on the top right. I made a lame photoshop version to show that they could be placed on the bottom right of the console.

This should in theory look  a bit cleaner and easier to spot.

![Panel](https://user-images.githubusercontent.com/64218536/175890398-f3914aed-5fe2-42d3-b2eb-2a0d9ccb50f3.png)

**Comment by the-w41k3r** [comment]
I personally feel that it looks more clean and modern, utilizing all the space on the side. And yes i think that the click copy ip adress should be added back.

**Comment by penderes** [comment]
I like this approach. I'd even say it looks less cluttered if you put the Load, Memory, etc. boxes below the respective graphs. Basically have Uptime and CPU Load below the CPU chart, Memory and Disk below the memory chart and Network I/O below the Network chart.

This also allows more space for the console.

The address box could be put next to the server name at the top.

Even more janky photoshop than yours to illustrate:

![image](https://user-images.githubusercontent.com/17273181/176109235-4248ca70-c533-4c38-95e5-5cb5488d91f2.png)

**Reply by snowmobile2004** [reply]
Another suggestion - user-configurable setting to use 1000 or 1024 as the base value? Then we can choose which one we'd prefer to use ourselves.
